lib/parser/file_group.ex

```diff
@@ -48,6 +55,27 @@ defmodule Thrift.Parser.FileGroup do
     end)
   end
 
+  def update_resolutions(file_group) do
+    # since in a file, we can refer to things defined in that file in a non-qualified
```

Comment:
I don't think so... 
"Since in a file, we can refer to things defined in that file in a non-qualified way..." makes sense to me.